api/guides/access_tokens.md

```diff
@@ -85,46 +79,27 @@ Next, upload the certificate (**crt**) file via the Adobe I/O configuration wind
 Multiple certificates may be added.
 {% endalert %}
 
-![](/images/access-token/IO-5.jpg)
-
-Next, select the **"Role"** or set of permissions for applications using this integration.
+![](/images/access-token/IO-5.png)
 
-* **Admin** - can do everything
-* **IT Admin** - can publish and manage properties, extensions, and environments
-* **Developer** - can only modify existing entities
-* **Approver** - can only modify and approve changes
-* **Publisher** - can modify entities, approve changes, and publish changes to production environments
-* **Extension Developer** - Can make changes and create new private and public extensions
+Next, we need to assign rights to this Integration.  These integrations share the same permissions that can be assigned to users through the Admin Console.  Any product profiles that exist for Launch will show up in the list at the bottom of the page.  Select the product profile that is appropriate for this integration.
 
-**DETAILED PERMISSION TABLE**
+If you need help creating a new product profile, an Org Admin or a Product Admin for Launch can do this is in the Admin Console where users are managed.  You can read more about this in the user docs [here](https://docs.adobe.com/content/help/en/launch/using/reference/admin/user-permissions.html#n3-create-your-product-profile).
 
-|                     | Admin | IT Admin | Developer | Approver | Publisher | Extension Developer |
-|---------------------|-------|----------|-----------|----------|-----------|---------------------|
-| Develop             |   X   |          |     X     |     X    |     X     |          X          |
-| Approve             |   X   |          |           |     X    |     X     |                     |
-| Publish             |   X   |     X    |           |          |     X     |                     |
-| Manage Properties   |   X   |     X    |           |          |           |                     |
-| Manage Extensions   |   X   |     X    |           |          |           |                     |
-| Manage Environments |   X   |     X    |           |          |           |                     |
-| Develop Extensions  |   X   |          |           |          |           |          X          |
+Click **"Create Integration"**.  You should now see the success screen.
 
-
-Click **"Create Integration"**
-
-![](/images/access-token/IO-6.jpg)
+![](/images/access-token/IO-6.png)
 
 {% alert info, Note %}
 If an error is encountered, it's generally related to certificate generation issues.
 {% endalert %}
 
-
 ## Testing the Integration
 
-Navigate to the Integration in the [Adobe I/O Console](https://console.adobe.io/).
+Click **Continue to integration details**.
```

Comment:
Don't have to do this now, but it might be a good idea to emphasize that testing the Integration is NOT for production use and users should be programmatically exchanging their tokens. I've had more than a few people ask me why their access token has expired and how to use the UI to generate a token etc.

Reply:
Is there a guide somewhere that we can point them to with instructions for how to do this programmatically inside their applications?